troubleshooting/debugging.md

```diff
@@ -674,6 +679,95 @@ To work around this issue, we suggest using the following option so that system
 net.dns.mode LEGACY
 ```
 
+### Runbooks
+
+
+#### FireLens Crash Report Runbook
+
+When you recieve a SIGSEGV/crash report from a FireLens customer, perform the following steps. 
+
+##### 1. Build and distribute a core dump S3 uploader image
```

Comment:
I wonder if we need details about using --platform and targeting the desired architecture when working on MacOS (building linux/amd64 images is not default). I am not sure the correct location to state that as make core will build whatever platform default and any guide information on building an image below may also need that note.